app/views/layouts/_flash.html.erb

```diff
@@ -0,0 +1,25 @@
+<% flash.each do |key, value| %>
```

Comment:
We've been re-using existing components from register so we don't rebuild things twice. Is there way we can re-use the flash component here? A bit of work was put it into it.

Reply:
https://github.com/DFE-Digital/register-trainee-teachers/blob/master/app/components/notification_banner/view.rb and https://github.com/DFE-Digital/register-trainee-teachers/blob/master/app/components/flash_banner/view.rb

Reply:
Added these in from register